src/components/Appointment/PatientCardInfoShort/index.tsx

```diff
@@ -11,36 +15,52 @@ import {
   PatientInfoName,
   UserInfo,
 } from '@components/Patient/styles';
-import {
-  cityCountryInfo,
-  emailInfo,
-  genderInfo,
-  nameInfo,
-  phoneNumberInfo,
-  yearsInfo,
-} from '@constants/mockData';
+import { patientApi } from 'services/PatientService';
+import { male, years } from '@constants/patient';
+import Spinner from '@components/Loaders/Spinner';
+
+function PatientCardInfo() {
+  const { id } = useParams();
+
+  const { data: patient, isLoading } = patientApi.useGetPatientByIdQuery(
+    Number(id)
+  );
 
-function PatientCardInfoShort() {
+  const patientFullName = `${patient?.firstName} ${patient?.lastName}`;
+  const patientCityCountry = `${patient?.city}, ${patient?.country}`;
+  const patientAge = `${
```

Comment:
you will recalculate it with each rerender - change

Reply:
fix. add useMemo